src/test/web/smoke.test/smoke.web.test.ts

```diff
@@ -18,9 +18,7 @@ suite('Web Extension Smoke Test Suite', () => {
     });
 
     test('Verify containers', () => {
-        const appShellSymbol = extensionApi.getSymbol<IApplicationShell>('IApplicationShell');
-        assert.ok(appShellSymbol, `Cannot get the symbol for IApplicationShell`);
-        const appShell = extensionApi.serviceManager?.get<IApplicationShell>(appShellSymbol!);
+        const appShell = extensionApi.serviceManager?.get<IApplicationShell>(IApplicationShell);
```

Comment:
Note; Previously we didn't get any intellisense in here at all, now we do.
This was because of an entry in `tsconfig.json`, now we get full code completion in these files.